### ISSUE TITLE
Investigate if anything should be done before an “execv” 

### ISSUE THREAD

**Issue by fopina** [surface-security/django-dkron#32]
Should we move this to a new issue? Or are we validating before `execv()`?

_Originally posted by @gsilvapt in https://github.com/surface-security/django-dkron/pull/30#discussion_r1141888082_
            
            
execv replaces the current process entirely so if there’s anything done by django after command.handle that should be done, we need to do it before calling it.

is there anything?

**Comment by tcppb** [comment]
close old issues
